src/wasm/literal.cpp

```diff
@@ -2270,14 +2273,20 @@ Literal Literal::geF64x2(const Literal& other) const {
                                                                     other);
 }
 
+static Literal PassThrough(const Literal& literal) { return literal; }
```

Comment:
I don't think we have a convention to use uppercase on such functions, unless I missed something?